src/configuration.rs

```diff
@@ -56,4 +56,64 @@ impl Configuration {
     }
 
     // TODO: implement builder pattern
+
+    pub fn dev_initial_reg_value(&self) -> u8 {
+        let dev_initial = match self.pwm_frequency {
+            PwmFrequency::Pwm62_5kHz => 0,
+            PwmFrequency::Pwm125kHz => BitFlags::DEV_INITIAL_PWM_FREQ,
+        } | self.data_ref_mode.register_value()
+            << BitFlags::DEV_INITIAL_DATA_REF_MODE_SHIFT
+            | (self.max_line_num & BitFlags::DEV_INITIAL_MAX_LINE_NUM_MASK)
+                << BitFlags::DEV_INITIAL_MAX_LINE_NUM_SHIFT;
+
+        dev_initial
+    }
+
+    pub fn dev_config1_reg_value(&self) -> u8 {
+        let dev_config1 = if self.cs_turn_on_delay {
```

Comment:
Looks like my changes to use `then_some(..).unwarp_or(0)` have gone missing here.

Reply:
Right.. will have a look.. I was working on this at the same time you pushed the support for cfg reg 2 and 3. So took your work and spliced it in there.. possibly missed something there.. :)